plugin.json

```
@@ -128,6 +128,12 @@
             "type": "bool",
             "help_text": "When true, synthetic users will be converted to members when they login for the first time.",
             "default": false
+        },{
+            "key": "metricsForSharedChannelsInfrastructure",
+            "display_name": "Add metrics Shared Channels Service (Not used yet)",
+            "type": "bool",
+            "help_text": "When true, the system collect metrics about Shared Channels Service usage by the plugin without using it.",
```

Propose avoiding the term "shared channels" in any user-facing setting. 0/5 on the actual term used, just trying to steer clear of the feature with the same name. I guess this configuration isn't long lived anyway, so we don't have to worry about a long term name.

```suggestion
            "key": "metricsForSharedChannelsInfrastructure",
            "display_name": "Initialize and monitor the durable queue service",
            "type": "bool",
            "help_text": "When true, the the durable queue service is enabled with metrics to evaluate the overhead. Message delivery is unaffected.",
```